dashboard/controller/classrooms/addClassroom.js

```diff
@@ -69,4 +67,4 @@ module.exports = function addClassroom(req, res) {
 			server: classroom.ini().information
 		});
 	}
-};
+};
```

Comment:
Add line break here and other files where a line break is missing at the end of the file in git diff on GitHub